scripts/bug_retriever.py

```diff
@@ -193,67 +193,59 @@ def retrieve_bugs(self, limit: int | None = None) -> None:
             bugzilla.download_bugs(inconsistent_bug_ids)
 
         # TODO: Figure out why.
-        missing_history_bug_ids = {
-            bug["id"]
-            for bug in bugzilla.get_bugs(
-                include_invalid=True,
-                include_additional_products=bugzilla.ADDITIONAL_PRODUCTS,
-            )
-            if "history" not in bug
-        }
-        bugzilla.delete_bugs(lambda bug: bug["id"] in missing_history_bug_ids)
-        logger.info(
-            "Deleted %d bugs as we couldn't retrieve their history",
-            len(missing_history_bug_ids),
-        )
+        handle_missing_field("history")
 
-        handle_missing_comments()
+        # TODO: Figure out why.
+        handle_missing_field("comments")
 
         zstd_compress(bugzilla.BUGS_DB)
 
 
-def handle_missing_comments(trial_number: int = 1, max_tries: int = 2) -> int:
-    """Handle bugs that are missing comments.
-
-    This function will try to re-download the bugs that are missing comments
-    and delete them if they are still missing comments after the maximum number
-    of tries.
+def handle_missing_field(
+    field_name: str, trial_number: int = 1, max_tries: int = 2
```

Comment:
nit: it would be faster if we supported a list of fields (so we iterate through bugs fewer times in total)

Reply:
Yes, I thought of that, but we would lose the detailed logging of the number of bugs for each case.

Reply:
Done in 6a637176a56066060adf0146ba81bbe76a0fb155